.github/ISSUE_TEMPLATE/user-story.md

```diff
@@ -0,0 +1,16 @@
+---
+name: User Story
+about: Tell us about how you are using KUDO, and get your logo on our website and
+  we'll send you some awesome custom swag !
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Name of your company
```

Comment:
Shall we ask for their name and position as well?

Reply:
Done